webui/master/src/containers/pages/Configuration/Configuration.test.tsx

```diff
@@ -57,5 +57,18 @@ describe('Configuration', () => {
     it('Matches snapshot', () => {
       expect(shallowWrapper).toMatchSnapshot();
     });
+
+    it('Searches data with valid search key', () => {
+      shallowWrapper.setState({ searchConfig: 'alluxio.' });
+      expect(shallowWrapper.find('filtered-data-body'));
```

Comment:
you'll need to add some condition at the end, otherwise i think this `expect` always returns true. when looking for an element, you can do something like `expect(shallowWrapper.find('filtered-data-body').length).toEqual(1);`